### ISSUE TITLE
[issue]: virustotal detects virus

### ISSUE THREAD

**Issue by blueskiesok** [ventoy/Ventoy#1703]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.71

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

Both

### Partition Style

MBR

### Disk Capacity

128gb

### Image file checksum (if applicable)

Yes.

### Image file download link (if applicable)

https://github.com/ventoy/Ventoy/releases/download/v1.0.71/ventoy-1.0.71-windows.zip

### What happened?

MaxSecure Trojan.Malware.300983.susgen
![image](https://user-images.githubusercontent.com/85849883/176572162-5a627310-15fe-4457-9b41-c22fd79b2c68.png)

**Comment by blueskiesok** [comment]
also detects three potential issues on 1.0.77
![image](https://user-images.githubusercontent.com/85849883/176572329-0bb720fd-cfb9-45e1-9898-2b76e0ae9ce3.png)

**Comment by ventoy** [comment]
It's false positive.  Ventoy is 100% open source.

**Comment by blueskiesok** [comment]
Was just letting you know.  That way you could possibly get it whitelisted. It is very helpful. 🤓

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by hedgehogdrfrink** [comment]
Hi just want to let you know this is still the case even with nordvpn.

**Comment by Ryland-w** [comment]
<img width="852" height="625" alt="Image" src="https://github.com/user-attachments/assets/34a7255d-c756-4c8b-88dc-9b93b7f38d96" />

**Comment by vibbe** [comment]
* The “Trojan.Malware.300983.susgen” detection comes from an antivirus engine (usually MaxSecure) that uses heuristic / behavior-based scanning, not a signature for a known malware.
[https://trojan-killer.net/maxsecure-trojan-malware-300983-susgen-guide/](url)

* Heuristic scanners tend to flag programs that do “unusual” or “risky-looking” behavior — especially installers, system utilities, tools that manipulate disks / partitions / boot sectors, or those that use custom packaging or compression, because those patterns resemble behaviors of actual malware.
[https://hackerdose.com/malware/trojan-malware-300983-susgen-false-positive/](url)

* Many other legitimate open-source tools (not just Ventoy) have gotten flagged the same way — e.g. BleachBit, WinMerge, restic, h5py have all had similar “susgen” detections from MaxSecure despite being clean.
[BleachBit issue 948](https://github.com/bleachbit/bleachbit/issues/948 #1950

*There are bunch of related topic on this matter too. Just look em up!*
 [](https://forums.malwarebytes.com/topic/283636-ventoy-installer-false-positive-detected-by-ai/)

- **Trojan Killer**  
  [Trojan.Malware.300983.Susgen False Positives Detection | Trojan Killer (9 April 2025)](https://trojan-killer.net/maxsecure-trojan-malware-300983-susgen-guide/)

- **HackerDose**  
  [Trojan.Malware.300983.susgen False Positive Threat Detection – HackerDose (4 October 2024)](https://hackerdose.com/malware/trojan-malware-300983-susgen-false-positive/)

- **GitHub**  
  [totalvirus detects Trojan.Malware.300983.susgen · Issue #948 · bleachbit/bleachbit](https://github.com/bleachbit/bleachbit/issues/948)

- **YouTube**  
  [Dont you worry 😂?](https://www.youtube.com/watch?v=dQw4w9WgXcQ)
  [False Positive Virus? or Should I Be Worried?](https://www.youtube.com/watch?v=wkK-Lm93gDw)

- **StackOverflow**  
  [How to stop antivirus false positives every time we re-release?](https://stackoverflow.com/questions/52360025/how-to-stop-antivirus-false-positives-everytime-we-re-release-software)

## Why Ventoy triggers this detection

Ventoy’s Windows installer (or portable zip) includes code that manipulates USB drives, partitions, boot sectors — operations that are similar to what malware or boot-sector infectors might do. That’s precisely the kind of behavior heuristic scanners treat as suspicious.

Historically, people using Ventoy have reported these false-positive detections. For example: the issue of “ventoy-1.0.81-windows.zip flagged as Trojan.Malware.300983.susgen” is documented. See discussions such as this thread on [Malwarebytes Forums](https://forums.malwarebytes.com/topic/283636-ventoy-installer-false-positive-detected-by-ai/). 

In forums and user-reports, multiple people note that the detection is by MaxSecure only and that other major antivirus engines (e.g. Malwarebytes) or Windows built-in protections do not detect anything suspicious — suggesting a false positive.

Some users mention that after extraction (or after installing), the detection disappears. For instance, in the same Malwarebytes discussion above, a commenter says they could not reproduce the detection when they manually extracted the installer’s contents.

In short: Ventoy’s technical functionality (disk / boot operations, custom executables, perhaps non-signed binaries) overlaps with what heuristic scanners consider “risky,” which triggers the “susgen” alert — even though Ventoy itself is open-source and widely used without widespread reports of malicious behavior.  


*Updated with some more links and text* *Please close this issue! @ventoy*
